cypress/specs/app.spec.ts

```diff
@@ -66,6 +82,11 @@ describe('App', () => {
     );
     cy.findByRole('tab', { name: 'Heatmap' }).should('exist');
     cy.findByRole('figure', { name: /twoD_cplx/ }).should('exist');
+
+    if (!!Cypress.env('TAKE_SNAPSHOTS')) {
+      cy.wait(SNAPSHOT_DELAY);
+      cy.matchImageSnapshot('heatmap_2D_complex');
+    }
```

Comment:
I added e2e tests for complex line and heatmap.

Do you think it is worth adding them for NeXus as well ?

Reply:
Neh, not worth it, I reckon. Did you add feature tests, though? Those are more important.

Reply:
This: https://github.com/silx-kit/h5web/pull/667/files#diff-217560973575044a39d13ed85c227f1cd30430e8db25666e0a013bd8399ab72f you mean ?